lsan_suppression.txt

```diff
@@ -0,0 +1,2 @@
+# Leak arises from OpenSSL usage of MoCOCrW and SoftHSM2.
```

Comment:
This is too brief to judge.
Please add a technical explanation why this leak cannot be solved and is accepted.

Reply:
It is not MoCOCwR code, as far as I am aware. Added more to the comment.